cli/grammar/src/main/congocc/nessie/nessie-cli.ccc

```diff
@@ -191,6 +194,69 @@ CreateNamespaceStatement
       ]
     ;
 
+RevertContentStatement
+    : <REVERT>
+      <CONTENT>
+      { setOptionalNextTokenTypes(TokenType.DRY); }
+      [ /dryRun/=<DRY>
```

Comment:
+1 to the dry run option in general, but doesn't `REVERT CONTENT DRY OF myKey ...` look a bit awkward? Why not `REVERT CONTENT OF ... DRY RUN`?

Reply:
b/c `MERGE` has `DRY` at the beginning as well